docs/changelog.txt

```diff
@@ -1,3 +1,26 @@
+0.5.1 (2020-05-10)
+------------------------------------------------------------------------
+Among other, debian packaging related reasons, this release firms up the number of allowed Road/Tramtypes.
+
+Roadtypes
```

Comment:
This release fixes a number of issues reported in 0.5.0.

N.B.: Although nmlc now permits up to 63 each of road and tramtypes, OpenTTD currently requires that the *combined* total not be more than 63 and that the same label not be used for both a roadtype and a tramtype. These limitations are not checked by nmlc.